niftypad/api/__init__.py

```diff
@@ -44,7 +59,8 @@ def kinetic_model(src, dst=None, params=None, model='srtmb_basis', w=None, r1=0.
     dt = np.asarray(meta['dt'])
     ref = np.asarray(meta['ref'])
     ref = Ref(ref, dt)
-    ref.interp_1cubic()
+    # change ref interpolation to selected method
+    ref.run_interp(input_interp_method=input_interp_method)
```

Comment:
Does this come with a list of the available methods?

Reply:
I think it's this:

https://github.com/AMYPAD/NiftyPAD/blob/75012a1636bed597ac70e3204faf8e86b969d9bf/niftypad/api/__init__.py#L21

Reply:
Yes as Casper said.